src/eterna/ui/SpecBox.ts

```diff
@@ -30,7 +30,7 @@ export default class SpecBox extends ContainerObject {
     protected added(): void {
         this._panel = new GamePanel({});
         if (!this._docked) {
-            this._panel.setup(0, 1.0, 0x152843, 0.27, 0xC0DCE7);
+            this._panel.setup(0, 1.0, 0x043468, 0.27, 0xC0DCE7);
```

Comment:
If the background color of this panel is changed, the button colors also need to be changed